tests/UI/campaigns/functional/BO/02_orders/02_invoices/03_invoiceOptions/06_otherOptions.js

```diff
@@ -148,4 +157,27 @@ describe('BO - Orders - Invoices : Update \'Invoice number and Footer text\'', a
       await expect(exist, `PDF does not contains this text : ${invoiceData.footerText}`).to.be.true;
     });
   });
+
+  describe('Back to the default data value', async () => {
+    it('should go to \'Orders > Invoices\' page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToInvoicesPageForDefaultData', baseContext);
+
+      await orderPageTabListBlock.goToSubMenu(
+        page,
+        orderPageTabListBlock.ordersParentLink,
+        orderPageTabListBlock.invoicesLink,
+      );
+
+      const pageTitle = await invoicesPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(invoicesPage.pageTitle);
+    });
+
+    it('should change the Invoice number, legal free text and the footer text to default data', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'backToDefaultData', baseContext);
+
+      await invoicesPage.clearInputOptions(page);
```

Comment:
You can use the `setInputOptions(page, data)` method with all data = null

Reply:
Done, thanks @nesrineabdmouleh ^^